corehq/apps/users/models.py

```diff
@@ -242,6 +242,9 @@ def normalize(self):
         if self.edit_apps:
             self.view_apps = True
 
+        if self.view_reports or bool(self.view_report_list):
+            self.download_reports = True
```

Comment:
This will prevent the `download_reports` permission for ever being set to `False` as long as the user also has `view_reports`.

I think this should likely be inverted:

```python
if not self.view_reports:
    self.download_reports = False
```

Reply:
ah ignore my comment above-- for new roles that should not be an issue and existing roles will be covered by the migration. Would I however need to add the permission to True [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/models.py#L343) where `view_reports=True`?

Reply:
I'm not sure if the default roles should be updated, seems like a question fir @dimagi/product